### ISSUE TITLE
Sync to browser account

### ISSUE THREAD

**Issue by WqyJh** [cnwangjie/better-onetab#355]
Chrome/Firefox provide `storage.sync` to sync extension data among multiple devices, which I think is a good choice for user.

According to chrome storage reference, the `storage.sync` behaves identically to `storage.local` in offline mode and would syncs the data when browser is online. Therefore, if all of the `storage.local` could be replaced by `storage.sync`, the data would automatically synced by browser as long as user logged in.

Even if the user won't log in, all of the data set by `storage.sync` would be stored locally, same with `storage.local`.

> When using storage.sync, the stored data will automatically be synced to any Chrome browser that the user is logged into, provided the user has sync enabled.
> 
> When Chrome is offline, Chrome stores the data locally. The next time the browser is online, Chrome syncs the data. Even if a user disables syncing, storage.sync will still work. In this case, it will behave identically to storage.local.
> 
> https://developer.chrome.com/docs/extensions/reference/storage/#usage

**Comment by PwrSrg** [comment]
Will **SYNC ever work**??  It has _never_ worked for me.
It just takes me to a dead site that times out.  https://boss.cnwangjie.com/login

**Comment by WqyJh** [comment]
No, it doesn't work at all. Seems this project is no longer under maintained.

**Comment by caiodev** [comment]
I fear you're right @WqyJh
